tests/test_particle_file.py

```diff
@@ -181,8 +181,9 @@ def IncrLon(particle, fieldset, time):
         particle.sample_var += 1.
         if particle.sample_var > fieldset.maxvar:
             particle.delete()
+
     for i in range(runtime):
-        pset.execute(IncrLon, dt=dt, runtime=1., output_file=pfile)
+        pset.execute(pset.Kernel(IncrLon, delete_cfiles=False), dt=dt, runtime=1., output_file=pfile)
```

Comment:
This change is not necessary anymore? Was only there for debugging purposes (keeping the cfiles)?

Reply:
agreed, thanks for the feedback, need to rewind that change